cmd/podman/system/service.go

```diff
@@ -131,20 +131,17 @@ func resolveAPIURI(_url []string) (string, error) {
 		if srvArgs.Varlink {
 			socketName = "io.podman"
 		}
-		socketDir := filepath.Join(xdg, "podman", socketName)
-		if _, err := os.Stat(filepath.Dir(socketDir)); err != nil {
```

Comment:
I don't think that'll fix https://github.com/containers/podman/issues/8184 entirely as it affects root (as well).

Reply:
root is handled below in the default case

Reply:
Time to move away from the screen :) Thanks!